components/o-typography/src/scss/use-cases/_general.scss

```
@@ -156,40 +146,17 @@
 		}
 	}
 
-	// Output external link styles.
-	// Fallback to a black icon if no link colour is given.
-	@if($external) {
-		$icon-color: if($base-color, $base-color, oColorsByName('black'));
-		$icon-size: 1em;
-		$icon-inbuilt-padding-size: #{div($icon-size, 4)};
-		$icon-padding-calc: "#{$icon-size} + #{0.5ch}";
-		@include oIconsContent('outside-page', $icon-color, $size: null, $include-base-styles: false);
-		background-repeat: no-repeat;
-		// Add padding for icon background.
-		// To position the icon closer to the copy, minus the right/left padding
-		// built into the icon svg (10px for a 40px icon)
-		// https://github.com/Financial-Times/fticons/blob/v1.20.5/contributing.md#design
-		padding-right: calc(#{$icon-padding-calc} - #{$icon-inbuilt-padding-size} - #{$icon-inbuilt-padding-size});
-		background-position-x: calc(100% + 0.5ch);
-		// Mimic baseline alignment.
-		@supports (background-size: contain) {
-			background-position-y: calc(100% + #{$underline-size} - 1px);
-		}
-		@supports (text-decoration-thickness: $underline-size) {
-			background-position-y: calc(100% + #{$icon-inbuilt-padding-size} - #{$underline-size} - 1px);
-		}
-		background-origin: border-box;
-		// IE11 does not size and position the svg icon background correctly
-		// using `background-size: contain`. Use @supports to exclude IE11,
-		// as IE11 does not support @supports
-		background-size: calc(#{$icon-padding-calc}) calc(#{$icon-padding-calc}); // IE11 fallback
-		@supports (background-size: contain) {
-			background-size: contain;
-		}
-		&[target="_blank"]:after {
-			@include oNormaliseVisuallyHidden();
-			content: '(opens a new window)';
-		}
+	@if ($external) {
+		@warn 'The $external parameter for oTypographyLink is deprecated. ' +
```

Nice, I like that you included why in the warning. I thought some explicit instruction might be helpful too
```suggestion
		@warn 'The $external parameter for oTypographyLink is deprecated. ' +
		'Remove the $external argument from the oTypographyLink call.' +
```